fleet-management-spring-mvc/src/main/webapp/WEB-INF/jsp/technician/vehicleDetailView.jsp

```diff
@@ -61,13 +61,12 @@
         </tr>
         </thead>
         <tbody>
-        <c:forEach items="${inspections}" var="inspection">
-        <tr>
-            <td>${inpection.name}</td>
-            <td>${inpection.days}</td>
-
-        </tr>
-    </c:forEach>
+            <c:forEach items="${inspections}" var="inspection">
+            <tr>
+                <td><c:out value="${inspection.name}"/></td>
```

Comment:
in old version of jsp you should always use c:out to output dynamic text, In this case ${inspection.name} doesn't work but c:out is working correct. I guess it is because of jsp version, but don't have time to check it deeper.